src/Extension/CoreExtension.php

```diff
@@ -266,6 +266,7 @@ public function getFilters(): array
             // iteration and runtime
             new TwigFilter('default', [self::class, 'default'], ['node_class' => DefaultFilter::class]),
             new TwigFilter('keys', [self::class, 'keys']),
+            new TwigFilter('invoke', [self::class, 'invoke']),
```

Comment:
The native function `call_user_func` would be identical.

```suggestion
            new TwigFilter('invoke', 'call_user_func'),
```

Reply:
In fact, no. It would allow calling any callable (including all PHP functions), not only Closure.